tests/ImageSharp.Interactive.Tests/FormatterTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.DotNet.Interactive.Formatting;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Drawing.Processing;
+using SixLabors.ImageSharp.Interactive;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing;
+using Xunit;
+
+namespace ImageSharp.Interactive.Tests
+{
+    public class FormatterTests : IDisposable
+    {
+        public FormatterTests() => KernelExtension.RegisterFormatters();
+
+        [Fact]
+        public async Task Formats_image_as_png()
+        {
+            var image = new Image<Rgb24>(100, 100);
+            string html = image.ToDisplayString(HtmlFormatter.MimeType);
+            var parser = new HtmlParser();
+            AngleSharp.Html.Dom.IHtmlDocument document = await parser.ParseDocumentAsync(html);
+            AngleSharp.Dom.IElement img = document.QuerySelector("img");
+            using var _ = new AssertionScope();
+
+            img.Should().NotBeNull();
+            img.Attributes["src"].Value.Should().Contain("data:image/png;base64,");
+        }
+
+        [Fact]
+        public async Task Formats_images_with_multiple_frames_as_gif()
+        {
+            var image = new Image<Rgba32>(400, 400);
```

Comment:
You can use the `new Image<Rgba32>(Configuration.Default, 400, 400, Color.Coral)` overload to create images with a background color. Using `Fill` like that is a per-pixel operation and, as such, is much slower (`Clear` is the better choice in ImageSharp.Drawing but you don't even need it here.)

Reply:
done